docs/pages/docs/avatar.mdx

```diff
@@ -23,10 +37,53 @@ An image element with a fallback for representing the user.
 ### `FAvatar(...)`
 
 ```dart
+// Default constructor - Valid image
 FAvatar(
-  image: const NetworkImage('https://picsum.photos/250?image=9'),
-  size: 60,
-  placeholderBuilder: (_) => const Text('MN'),
+  image: const NetworkImage('https://raw.githubusercontent.com/forus-labs/forui/main/samples/assets/avatar.png'),
+  placeholder: const Text('MN'),
+),
+
+// Default constructor - Invalid image with placeholder
+FAvatar(
+  image: const NetworkImage(''),
+  placeholder: const Text('MN'),
+),
+
+// Default constructor - Invalid image without placeholder
+FAvatar.raw(
+  image: const NetworkImage(''),
 ),
 ```
 
+### `FAvatar.raw(...)`
```

Comment:
I'm honestly a bit confused about the layout here. Is this supposed to be under Examples?

If so, please add the Examples header and maybe give it a more descriptive name.